app/models/server/models/Users.js

```diff
@@ -757,6 +757,19 @@ export class Users extends Base {
 		return this.find({ active: true, isRemote: true }, options);
 	}
 
+	findOneByIdAndSAMLProvider(_id, provider) {
+		return this.findOne({
+			_id,
+			'services.saml.provider': provider,
+		}, {
+			'services.saml': 1,
+		});
+	}
+
+	findOneByEppn(eppn) {
```

Comment:
Can't find where this was used

Reply:
yes, good catch.. it was part of another fix